gix-fs/src/symlink.rs

```diff
@@ -59,7 +59,7 @@ pub fn create(original: &Path, link: &Path) -> io::Result<()> {
 pub fn is_collision_error(err: &std::io::Error) -> bool {
     // TODO: use ::IsDirectory as well when stabilized instead of raw_os_error(), and ::FileSystemLoop respectively
     err.kind() == AlreadyExists
-            || err.raw_os_error() == Some(21)
+            || err.raw_os_error() == Some(if cfg!(windows) { 5 } else { 21 })
```

Comment:
This function here appears to be under `#[cfg(not(windows))]` making `cfg!(windows)` always false.
The `#[cfg(windows)]` below should probably be changed instead.

Reply:
That's a great catch, thank you!

I pushed the fix [into this standing PR](https://github.com/Byron/gitoxide/pull/1368).